system/jlib/jptree.hpp

```diff
@@ -239,6 +239,7 @@ jlib_decl void addPTreeItem(IPropertyTree *ptree, const char * name, const char
 
 jlib_decl IPropertyTree *createPTree(MemoryBuffer &src, byte flags=ipt_none);
 jlib_decl IPropertyTree *createPTree(IBufferedSerialInputStream &src, byte flags=ipt_none);
+jlib_decl IPropertyTree *createPTree(IBufferedSerialInputStream &src, const char *tag, IPTreeNodeCreator *nodeCreator=nullptr);
```

Comment:
you should not have to pass the tag. The root tag should be part of the stream.
Same as other deserializors - this shouldn't be an exception/special.

Reply:
In chat we agreed on calling this createPTreeFromBinary I think.